src/test/url-handling.test.js

```diff
@@ -240,8 +240,9 @@ describe('url handling tracks', function() {
     });
 
     it('will not accept invalid tracks in the thread order', function() {
-      const { getState } = initWithSearchParams('?globalTrackOrder=10');
-      expect(urlStateSelectors.getGlobalTrackOrder(getState())).toEqual([1, 0]);
+      const { getState } = initWithSearchParams('?globalTrackOrder=102');
+      // This will result in being the default order.
+      expect(urlStateSelectors.getGlobalTrackOrder(getState())).toEqual([0, 1]);
     });
```

Comment:
This test wasn't doing what it was supposed to do for a long time... took the opportunity to fix it here.